include/libopencm3/stm32/h7/rcc.h

```diff
@@ -0,0 +1,458 @@
+  RCC_TIM13       = _REG_BIT(0xE8, 7),
+  RCC_TIM14       = _REG_BIT(0xE8, 8),
+  RCC_LPTIM1      = _REG_BIT(0xE8, 9),
+  RCC_SPI2        = _REG_BIT(0xE8, 14),
+  RCC_SPI3        = _REG_BIT(0xE8, 15),
+  RCC_SPDIFRX     = _REG_BIT(0xE8, 16),
+  RCC_USART2      = _REG_BIT(0xE8, 17),
+  RCC_USART3      = _REG_BIT(0xE8, 18),
+  RCC_UART4       = _REG_BIT(0xE8, 19),
+  RCC_UART5       = _REG_BIT(0xE8, 20),
+  RCC_I2C1        = _REG_BIT(0xE8, 21),
+  RCC_I2C2        = _REG_BIT(0xE8, 22),
+  RCC_I2C3        = _REG_BIT(0xE8, 23),
+  RCC_CEC         = _REG_BIT(0xE8, 27),
+  RCC_DAC         = _REG_BIT(0xE8, 29),
+  RCC_UART7       = _REG_BIT(0xE8, 30),
+  RCC_UART8       = _REG_BIT(0xE8, 31),
+
+  /* APB1H peripherals*/
+  RCC_CRS         = _REG_BIT(0xEC, 1),
+  RCC_SWP         = _REG_BIT(0xEC, 2),
+  RCC_OPAMP       = _REG_BIT(0xEC, 4),
+  RCC_MDIO        = _REG_BIT(0xEC, 5),
+  RCC_FDCAN       = _REG_BIT(0xEC, 8),
+
+  /* APB2 peripherals */
+  RCC_TIM1        = _REG_BIT(0xF0, 0),
+  RCC_TIM8        = _REG_BIT(0xF0, 1),
+  RCC_USART1      = _REG_BIT(0xF0, 4),
+  RCC_USART6      = _REG_BIT(0xF0, 5),
+  RCC_SPI1        = _REG_BIT(0xF0, 12),
+  RCC_SPI4        = _REG_BIT(0xF0, 13),
+  RCC_TIM15       = _REG_BIT(0xF0, 16),
+  RCC_TIM16       = _REG_BIT(0xF0, 17),
+  RCC_TIM17       = _REG_BIT(0xF0, 18),
+  RCC_SPI5        = _REG_BIT(0xF0, 20),
+  RCC_SAI1        = _REG_BIT(0xF0, 22),
+  RCC_SAI2        = _REG_BIT(0xF0, 23),
+  RCC_SAI3        = _REG_BIT(0xF0, 24),
+  RCC_DFSDM       = _REG_BIT(0xF0, 28),
+  RCC_HRTIM       = _REG_BIT(0xF0, 29),
+
+  /* APB3 peripherals */
+  RCC_LTDCEN      = _REG_BIT(0xE4, 3),
+  RCC_WWDG1EN     = _REG_BIT(0xE4, 6),
+
+  /* APB4 peripherals */
+  RCC_SYSCFG      = _REG_BIT(0xF4, 1),
+  RCC_LPUART1     = _REG_BIT(0xF4, 3),
+  RCC_SPI6        = _REG_BIT(0xF4, 5),
+  RCC_I2C4        = _REG_BIT(0xF4, 7),
+  RCC_LPTIM2      = _REG_BIT(0xF4, 9),
+  RCC_LPTIM3      = _REG_BIT(0xF4, 10),
+  RCC_LPTIM4      = _REG_BIT(0xF4, 11),
+  RCC_LPTIM5      = _REG_BIT(0xF4, 12),
+  RCC_COMP12      = _REG_BIT(0xF4, 14),
+  RCC_VREF        = _REG_BIT(0xF4, 15),
+  RCC_RTCAPB      = _REG_BIT(0xF4, 16),
+  RCC_SAI4        = _REG_BIT(0xF4, 21),
+};
+
+enum rcc_periph_rst {
+   /* AHB1 peripherals */
+  RST_DMA1        = _REG_BIT(0x80, 0),
+  RST_DMA2        = _REG_BIT(0x80, 1),
+  RST_ADC12       = _REG_BIT(0x80, 5),
+  RST_ETH1MAC     = _REG_BIT(0x80, 15),
+  RST_USB1OTGRST  = _REG_BIT(0x80, 25),
+  RST_USB2OTGRST  = _REG_BIT(0x80, 26),
+
+  /* AHB2 peripherals */
+  RST_DCMI        = _REG_BIT(0xDC, 0),
+  RST_CRYP        = _REG_BIT(0xDC, 4),
+  RST_HASH        = _REG_BIT(0xDC, 5),
+  RST_RNG         = _REG_BIT(0xDC, 6),
+  RST_SDMMC2      = _REG_BIT(0xDC, 9),
+
+  /* AHB3 peripherals */
+  RST_MDMA        = _REG_BIT(0x7C, 0),
+  RST_DMA2D       = _REG_BIT(0x7C, 4),
+  RST_JPGDEC      = _REG_BIT(0x7C, 5),
+  RST_FMC         = _REG_BIT(0x7C, 12),
+  RST_QSPI        = _REG_BIT(0x7C, 14),
+  RST_SDMMC1      = _REG_BIT(0x7C, 16),
```

Comment:
there's a bit 31, CPURST in my doc, should we not include it?

Reply:
![image](https://user-images.githubusercontent.com/2933763/69820096-5548f000-11ce-11ea-88d6-f232bdc0d451.png)